bgpd/bgp_vty.c

```diff
@@ -11520,6 +11520,82 @@ DEFPY (show_bgp_vrfs,
 	return CMD_SUCCESS;
 }
 
+DEFPY(show_bgp_router,
+      show_bgp_router_cmd,
+      "show bgp router [json]",
+      SHOW_STR
+      BGP_STR
+      "Overall BGP information\n"
+      JSON_STR)
+{
+	char timebuf[MONOTIME_STRLEN];
+	u_char uj = use_json(argc, argv);
+	json_object *json = NULL;
+
+	if (uj)
+		json = json_object_new_object();
+
+	time_to_string(bm->start_time, timebuf);
+
+	if (uj) {
+		json_object_string_add(json, "bgpStartedAt", timebuf);
```

Comment:
IMO, we should add just a UNIX timestamp here, otherwise, it's a mess for non-human stuff (automation, etc) to work easily...

Reply:
Changed!